src/static/index.js

```diff
@@ -0,0 +1,80 @@
+function handleEditClick(event) {
```

Comment:
Was the motivation for this script not wanting to reload the entire screen because it's a bad experience for users? I agree that reloading the entire screen is a clunky experience that we should avoid. The the book linked to in the roadmap issue addresses this issue. I explained this poorly, but I was thinking we could leverage the library that book mentioned (https://htmx.org/) to create a better user experience.

I was thinking we could create a "web 1.0" app, then add in HTMX to provide a better user experience similar to the approach taken in [hypermedia.systems](https://hypermedia.systems). What are your thoughts on this?